### PR TITLE
Deprecates python2

### PR DESCRIPTION
**Pull request by pedrocamargo**

### CODE REVIEW

File: fasttrips/Stop.py [hunk 1]
```diff
@@ -162,9 +162,9 @@ def add_daps_tazs_to_stops(self, dap_df, dap_id_colname, taz_df, taz_id_colname)
 
         # make sure the DAP IDs are unique from Stop IDs
         daps_unique_df  = dap_df.drop_duplicates().reset_index(drop=True)
-        join_daps_stops = pd.merge(left=daps_unique_df, right=self.stop_id_df,
-                                       how="left",
-                                       left_on=dap_id_colname,  right_on=Stop.STOPS_COLUMN_STOP_ID)
+        # join_daps_stops = pd.merge(left=daps_unique_df, right=self.stop_id_df,
```
Comment (janzill) [inline fasttrips/Stop.py:166]: unused variable, I double-checked :)

File: tests/test_cost_symmetry.py [hunk 2]
```diff
@@ -161,6 +161,7 @@ def test_cost_symmetry(pathfinder_paths, simulation_paths):
     assert pf_pathset_paths.shape[0] == paths.shape[0]
     assert pf_pathset_links.shape[0] == links.shape[0]
 
+
```
Comment (janzill) [inline tests/test_cost_symmetry.py]: linting would tell you off for adding empty lines :)

File: tests/test_weight_qualifiers.py [hunk 3]
```diff
@@ -87,6 +87,7 @@ def test_parse_weight_qualifiers_bad_key(sample_dataframe):
     sample_dataframe.loc[sample_dataframe[PathSet.WEIGHTS_COLUMN_WEIGHT_NAME].str.contains('logistic') , 'weight_name'] = \
         sample_dataframe[PathSet.WEIGHTS_COLUMN_WEIGHT_NAME].str.replace('logistic', 'logging')
 
+
```
Comment (janzill) [inline tests/test_weight_qualifiers.py]: empty line

File: fasttrips/Stop.py [hunk 4]
```diff
@@ -207,8 +207,9 @@ def add_daps_tazs_to_stops(self, dap_df, dap_id_colname, taz_df, taz_id_colname)
 
         # write the stop ids and zone ids to numbering file
         stop_id_df = self.stop_id_df  # local copy with filled NA
-        if Stop.STOPS_COLUMN_ZONE_ID_NUM not in stop_id_df.columns.values:
-            stop_id_df[Stop.STOPS_COLUMN_ZONE_ID_NUM] = -1
+        for col in [Stop.STOPS_COLUMN_ZONE_ID_NUM, Stop.STOPS_COLUMN_ZONE_ID]:
```
Comment (janzill) [inline fasttrips/Stop.py:211]: This looks like you added STOPS_COLUMN_ZONE_ID to the list, is that on purpose?

File: tests/test_maxStopProcessCount.py [hunk 5]
```diff
@@ -3,12 +3,12 @@
 import pytest
 from fasttrips import Run
 
-EXAMPLE_DIR    = os.path.join(os.getcwd(), 'fasttrips', 'Examples', 'Springfield')
```
Comment (janzill) [inline tests/test_maxStopProcessCount.py:6]: It looks like the changes to the Springfield 'A' config are not part of this (increasing cost_to_util), but I think we should keep these two changes together for clarity

**Review by janzill — COMMENTED** [review]
One question and I think the cost_to_util update for the Springfield example is missing

**Comment by janzill** [comment]
@pedrocamargo I just realised that the test updates are on master already, I just didn't see them yet.

File: tests/test_maxStopProcessCount.py (hunk 5 above)
Comment (janzill) [reply tests/test_maxStopProcessCount.py:6]: And they are on master already, my bad.

File: fasttrips/Stop.py (hunk 4 above)
Comment (pedrocamargo) [reply fasttrips/Stop.py:211]: Hmmm I remember it failing tests when you don't. Let me try again.

Comment (pedrocamargo) [reply fasttrips/Stop.py:211]: Confirmed. This is needed in order to fix test_bunny

Comment (janzill) [reply fasttrips/Stop.py:211]: Hm, four lines down nans are filled with 'None', maybe that's the right way to go here too then?

Comment (pedrocamargo) [reply fasttrips/Stop.py:211]: I replaced -1 with NaN, so the behaviour from the line you mentioned is preserved

Comment (janzill) [reply fasttrips/Stop.py:211]: My first thought was "that's not right", but then I realised they will be filled in the next step, so all good :)

**Review by janzill — APPROVED** [review]
